src/main/java/seedu/address/logic/Messages.java

```diff
@@ -55,6 +55,8 @@ public static String format(Person person) {
                 .append(person.getStudentId())
                 .append("; Tags: ");
         person.getTags().forEach(builder::append);
+        builder.append("; Class: ")
+                .append(person.getClassroom());
```

Comment:
change to person.getFormClass() instead of person.getClassroom

Reply:
fixed